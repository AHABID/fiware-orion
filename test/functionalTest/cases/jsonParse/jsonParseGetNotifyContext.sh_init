source ../harnessFunctions.sh

dbInit CB
brokerStart CB

