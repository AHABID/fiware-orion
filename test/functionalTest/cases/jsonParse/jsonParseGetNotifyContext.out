1: ***********************************
HTTP/1.1 405 Method Not Allowed
Content-Length: 0
Allow: POST
Date: Tue, 03 Jun 2014 08:55:43 GMT

